src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -2172,8 +2172,9 @@ public void setShowChatTime(final boolean showTime) {
   }
 
   public Optional<InGameLobbyWatcherWrapper> getInGameLobbyWatcher() {
-    if( ServerGame.class.isAssignableFrom(getGame().getClass())) {
-      return Optional.of(((ServerGame) getGame()).getInGameLobbyWatcher());
+    if(ServerGame.class.isAssignableFrom(getGame().getClass())) {
+      ServerGame serverGame = (ServerGame) getGame();
+      return Optional.ofNullable(serverGame.getInGameLobbyWatcher());
```

Comment:
Fix here, note the `ofNullable`, 'getInGameLobbyWatcher' will return null in single player games.